lazyflow/metaDict.py

```diff
@@ -40,6 +40,9 @@ def __init__(self, other=None, *args, **kwargs):
             # are ready
             self._ready = False
 
+        if not 'has_mask' in self:
+            self.has_mask = False
+
```

Comment:
Very minor change needed here: In the `MetaDict` class, all attributes that are not present in the dict are already returned as `None` if someone attempts to access them.  Since `has_mask` only applies to array data, and it defaults to `False` anyway, I think it would be easier to simply avoid initializing it with a value here.  Just delete these two lines.  Then `has_mask` will be implemented in the same way as other array-like slot metadata (e.g. `shape`, `dtype`, `axistags`, etc.)